docker_deploy/README.md

```diff
@@ -0,0 +1,26 @@
+# Deploy on Kubernetes using Docker containers
+
+In our environment it was `PROJECT_DIR=$HOME/git-repos/osv-microservice-demo`.
+
+## UI
+
+Build container using the provided Dockerfile-ui:
+```bash
+$ cd $PROJECT_DIR
+$ npm install
+$ docker build -t mike/osv-microservice-demo . -f docker_deploy/Dockerfile-ui
```

Comment:
Can I immediately start the container out of what we've just built? If so, I suggest to add a docker cmd that will run the UI container.

Then proceed with the k8s docs below.